docs/team/yixiann.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page title: Tan Yi Xian's Project Portfolio Page
```

Comment:
Typo in front matter. This will break the rendering. Reverting this PR for now via https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/37.

```suggestion
layout: page
title: Tan Yi Xian's Project Portfolio Page
```